apps/eas-expo-go/eas.json

```diff
@@ -91,6 +71,39 @@
         "simulator": true,
         "buildConfiguration": "Release"
       }
+    },
+    "release-client": {
+      "extends": "versioned-client",
+      "env": {
+        "EAS_BUILD_PROFILE": "release-client"
+      },
+      "android": {
+        "cache": {
+          "disabled": true
+        },
+        "withoutCredentials": false,
+        "gradleCommand": ":app:bundleVersionedRelease"
+      },
+      "ios": {
+        "cache": {
+          "disabled": true
+        },
+        "simulator": false
+      }
+    }
+  },
+  "submit": {
+    "release-client": {
+      "ios": {
+        "ascApiKeyPath": "./credentials/asc_api_key.p8",
+        "ascApiKeyId": "GT3KPQKKBJ",
+        "ascApiKeyIssuerId": "69a6de96-160a-47e3-e053-5b8c7c11a4d1",
+        "ascAppId": "982107779"
+      },
+      "android": {
+        "serviceAccountKeyPath": "./credentials/api-8143610498327843040-246301-25839a6ccc53.json",
+        "track": "internal"
+      }
```

Comment:
The other places you mentioned are relatively easy to change because they are in js, but here we would need to replace the file for a bit and restore it after. It's fine to do it if we need it, but my understanding is that all those ios values are harmless, and for Android I can change the path to sth without any ids.

Reply:
thanks for following up on those comments. they are all about nit comments, it's not truly critical. please only address the comments only when those comments make sense to you. since the pr is about submitting expo-go, my idea is just that the less information we expose publicly, we increase more security.

all changes look good to me now. i'm approving this pr. if there are any other further changes, will take a look again.